dialogflow/cloud-client/src/main/java/com/example/dialogflow/DetectIntentAudio.java

```diff
@@ -27,33 +27,19 @@
 import com.google.cloud.dialogflow.v2.SessionName;
 import com.google.cloud.dialogflow.v2.SessionsClient;
 import com.google.protobuf.ByteString;
+
+import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 
-
-/**
- * DialogFlow API Detect Intent sample with audio files.
- */
 public class DetectIntentAudio {
-  // [START dialogflow_detect_intent_audio]
 
-  /**
-   * Returns the result of detect intent with an audio file as input.
-   *
-   * Using the same `session_id` between requests allows continuation of the conversation.
-   *
-   * @param projectId     Project/Agent Id.
-   * @param audioFilePath Path to the audio file.
-   * @param sessionId     Identifier of the DetectIntent session.
-   * @param languageCode  Language code of the query.
-   * @return QueryResult for the request.
-   */
+  // DialogFlow API Detect Intent sample with audio files.
   public static QueryResult detectIntentAudio(
-      String projectId,
-      String audioFilePath,
-      String sessionId,
-      String languageCode)
-      throws Exception {
+          String projectId,
+          String audioFilePath,
+          String sessionId,
+          String languageCode) throws IOException {
```

Comment:
Looking at the code in googleapis/java-dialogflow searching for `throws` and `catch`, I see lots of specific enhanced exceptions - our samples should reflect the richness of those.  For example, [`ApiException`](http://googleapis.github.io/gax-java/1.7.1/apidocs/com/google/api/gax/rpc/ApiException.html) has 16 known subclasses that Gax throws.  gRPC also throws `io.grpc.StatusException` which has additional info to help developers understand the cause of their errors.  There is also `io.grpc.StatusRuntimeException` and `io.grpc.ManagedChannelProvider.ProviderNotFoundException`.